RELEASE.md

```diff
@@ -1,8 +1,14 @@
-# _Infrastructure - Dependency Update_
```

Comment:
@eteridvalishvili @TheJuanAndOnly99 - not sure why this file needs to be changed at all.

the `RELEASE.md` file hosts content related with CDM releases, and it's only touched a) by CDM maintainers and b) during a release process.

Please revert any change to this file. Thank you!

Reply:
I reverted the changes!